sdk/src/test/scala/com/horizen/api/http/SidechainApiRouteTest.scala

```diff
@@ -124,10 +125,12 @@ abstract class SidechainApiRouteTest extends AnyWordSpec with Matchers with Scal
   val mainchainBlockReferenceInfoRef = utilMocks.mainchainBlockReferenceInfoRef
   val keyRotationProof = utilMocks.keyRotationProof
   val certifiersKeys = utilMocks.certifiersKeys
+  val credentials = HttpCredentials.createBasicHttpCredentials("username","password")
+  val badCredentials = HttpCredentials.createBasicHttpCredentials("username","wrong_password")
 
   val mockedRESTSettings: RESTApiSettings = mock[RESTApiSettings]
   Mockito.when(mockedRESTSettings.timeout).thenAnswer(_ => 1 seconds)
-  Mockito.when(mockedRESTSettings.apiKeyHash).thenAnswer(_ => Some("aa8ed2a907753a4a7c66f2aa1d48a0a74d4fde9a6ef34bae96a86dcd7800af98"))
+  Mockito.when(mockedRESTSettings.apiKeyHash).thenAnswer(_ => Some("$2a$10$O5FAloC/gNuwpeoVKiV41.EcIlpOlk5hzsqYpleSmOEEKgj0j7BX6"))
```

Comment:
instead of having a magic value, let's use `BCrypt.hashpw` method with passing `credentials` data there.

Reply:
Done